db/migrate/20161102045612_add_resolve_url_option_to_website_agent.rb

```diff
@@ -0,0 +1,19 @@
+class AddResolveUrlOptionToWebsiteAgent < ActiveRecord::Migration[5.0]
+  def up
+    Agents::WebsiteAgent.find_each do |agent|
+      keys = agent.event_keys
+      if keys.nil? || keys.include?('url')
```

Comment:
In full JSON mode, there's no `extract` and you have no idea how each event looks like and you can't tell whether the JSON structure doesn't contain a `url` key, so this migration turns on `resolve_url` just in case.